lib/manageiq/environment.rb

```diff
@@ -9,16 +9,16 @@ def self.manageiq_plugin_setup(plugin_root = nil)
       # determine plugin root dir. Assume we are called from a 'bin/' script in the plugin root
       plugin_root ||= Pathname.new(caller_locations.last.absolute_path).dirname.parent
 
-      manageiq_plugin_update(plugin_root)
+      manageiq_plugin_update(plugin_root, force: false)
     end
 
-    def self.manageiq_plugin_update(plugin_root = nil)
+    def self.manageiq_plugin_update(plugin_root = nil, force: true)
```

Comment:
`force` here as a variable name has less meaning as it's out of context.  Thinking maybe `force_bundle_update`